backend/services/iot-manager/client/client.go

```diff
@@ -50,6 +52,10 @@ func New() *http.Client {
 }
 
 func addrIsGlobalUnicast(network, address string, _ syscall.RawConn) error {
+	c := config.Config
+	if c.GetBool(dconfig.SettingDomainSkipVerify) {
+		return nil
+	}
```

Comment:
Just to be a good lazy citizen, can we load this config once?
```suggestion
var (
	skipVerify         bool
	skipVerifyLoadOnce sync.Once
)

func addrIsGlobalUnicast(network, address string, _ syscall.RawConn) error {
	skipVerifyLoadOnce.Do(func() {skipVerify := config.Config.GetBool(dconfig.SettingDomainSkipVerify})
	if skipVerify {
		return nil
	}
```

Reply:
sure! I just followed what I found in the rest of the backend codebase 🙈